cmd2/utils.py

```diff
@@ -350,6 +350,17 @@ def isatty(self) -> bool:
         else:
             return False
 
+    @property
+    def line_buffering(self) -> bool:
+        """
```

Comment:
Consider adding a unit test or two to make sure pith positive and negative cases of this method are cover d.